src/ui/Facet/Facet.ts

```diff
@@ -604,8 +604,11 @@ export class Facet extends Component {
      * framework uses the last occurrence of the option.
      *
      * Default value is `800`.
+     *
+     * @deprecated This option is exposed for legacy reasons, and the recommendation is to not use this option. See
+     * {@link SearchInterface.responsiveComponents} instead.
      */
-    responsiveBreakpoint: ComponentOptions.buildNumberOption({ defaultValue: 800 }),
+    responsiveBreakpoint: ComponentOptions.buildNumberOption({ defaultValue: 800, deprecated: 'This option is exposed for legacy reasons, and the recommendation is to not use this option.' }),
```

Comment:
I would also go as far as completely removing the JSDOC for deprecated doc. It serves pretty much no purposes, and only pollutes the doc website with useless info.